### DISCUSSION TITLE
[FEATURE-REQUEST] La visualización de una rutina puede quedar ampliada, en vez que aumente a medida que se agreguen pb?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#241] [3. Ideas & Feature Requests] [answered]
Hola mi consulta es que si puede quedar la vista / visualización de una rutina en formato ya expandido y no por expandir. Lo pregunto porque me di cuenta que existen ahora 3 barras verticales y confunde un poco cual es cual, como tambien que cuando quieras subir, se sube la otra.

Adjunto un SS.

![image](https://user-images.githubusercontent.com/61419471/121043225-6dfe7280-c782-11eb-85d3-e3228c06d59f.png)

Gracias

**Comment by guillevalin** [comment, marked as answer]
Gracias @NicolasPLG por el feedback, efectivamente es una buena sugerencia que podría ayudar a facilitar la implementación de rutinas.

Sería bueno considerarlo en el roadmap de UI/UX @acedward .
